site/src/util/zipGeocoding.ts

```diff
@@ -0,0 +1,46 @@
+import { StateCode, USZipCode } from "../components/address-input/SmartyStreets";
+
+const smartyApiURL = "https://us-zipcode.api.smartystreets.com/lookup";
+const smartyAuthId = (process.env.SMARTY_STREETS_KEY as string);
```

Comment:
Wasn't sure about this, but maybe this can save us passing the API key around as a prop?

Reply:
Should be passed in as an argument to this function

Reply:
Happy to change it back, but can you help me understand why? Is the API key going to change such that we need to be able to inject it at each call site? Seems like this ways saves us a lot of param passing...

Reply:
Just good practice. We don't know how the business requirements will change in the future. If a partner wants to use their own API key this would become a bug we'd have to track down.

Reply:
Not sure I agree, seems like an unlikely case that a partner would go through the trouble of adding their own API key (versus the likelihood of a bug from threading the API key param through the various levels of components), but went back to passing the param for the time being.